keel-ec2/src/main/kotlin/com/netflix/spinnaker/keel/ec2/asset/AmazonSecurityGroupHandler.kt

```diff
@@ -99,9 +99,9 @@ class AmazonSecurityGroupHandler(
             "vpcId" to spec.vpcName,
             "description" to spec.description,
             "securityGroupIngress" to portRangeRuleToJob(spec),
-            "ipIngress" to spec.inboundRuleList.filter { it.hasCidrRule() }.flatMap {
-              convertCidrRuleToJob(it)
-            },
+            "ipIngress" to spec.inboundRuleList
+              .filter { it.hasCidrRule() }
+              .flatMap { convertCidrRuleToJob(it) },
```

Comment:
Hopefully this makes sense. This is the part where it's creating the actual change via River.